scenarios/sslstream.benchmarks.yml

```diff
@@ -0,0 +1,89 @@
+# Example:
+# crank --config https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/sslstream.benchmarks.yml --scenario read-write --profile aspnet-perf-lin
+
+imports:
+  - https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/aspnet.profiles.standard.yml
+
+variables:
+  # common parameters
+  serverPort: 5000
+  sendBufferSize: 32768 # bytes
+  receiveBufferSize: 32768 # bytes
+  tlsVersion: "1.3" # <1.2|1.3>
+  serverCertSelection: "CertContext" # <Callback|Certificate|CertContext>
+  clientCertSelection: "CertContext" # <Callback|Collection|CertContext>
+
+  # client-specific parameters
+  warmup: 15 #seconds
+  duration: 15 #seconds
+
+  # server-specific parameters
+  # requireClientCert: false
+
+scenarios:
+  read-write:
+    server:
+      job: sslServer
+      agent: secondary
+    client:
+      job: sslClient
+      agent: main
+      variables:
+        scenario: ReadWrite
+        host: "{{secondaryAddress}}"
+
+  client-read:
+    server:
+      job: sslServer
+      agent: secondary
+      variables:
+        receiveBufferSize: 0
+    client:
+      job: sslClient
+      agent: main
+      variables:
+        scenario: ReadWrite
+        sendBufferSize: 0

```

Comment:
I think this and `client-write` scenarios should also include `host: "{{secondaryAddress}}"`, shouldn't they? 🤔